display_information/sketch-on-map/src/test/java/com/esri/samples/sketch_on_map/SketchOnMapTest.java

```diff
@@ -0,0 +1,351 @@
+package com.esri.samples.sketch_on_map;
+
+import com.esri.arcgisruntime.geometry.Geometry;
+import com.esri.arcgisruntime.geometry.GeometryType;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.util.ListenableList;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Point2D;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.input.MouseButton;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import org.junit.Before;
+import org.junit.Test;
+import org.testfx.framework.junit.ApplicationTest;
+import org.testfx.robot.Motion;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.fail;
+
+public class SketchOnMapTest extends ApplicationTest {
+
+  MapView mapView;
+  GraphicsOverlay graphicsOverlay;
+
+  Button editButton;
+  Button stopButton;
+
+  Button pointButton;
+  Button multipointButton;
+  Button polylineButton;
+  Button polygonButton;
+  Button freehandPolylineButton;
+  Button freehandPolygonButton;
+
+  Button undoButton;
+  Button redoButton;
+  Button saveButton;
+  Button clearButton;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+    // set up the scene
+    FXMLLoader loader = new FXMLLoader(getClass().getResource("/sketch_on_map.fxml"));
+    Parent root = loader.load();
+    Scene scene = new Scene(root);
+
+    stage.setScene(scene);
+    stage.setTitle("Sketch on Map Sample");
+    stage.setWidth(1000);
+    stage.setHeight(700);
+    stage.show();
+
+    // wait for initialization
+    sleep(1500);
+  }
+
+  @Before
+  public void setup() {
+
+    // get a handle on the MapView and GraphicsOverlay
+    Node mapViewNode = lookup("#mapView").query();
+    if (mapViewNode instanceof MapView) {
+      mapView = (MapView) mapViewNode;
+      graphicsOverlay = mapView.getGraphicsOverlays().get(0);
+
+    } else {
+      fail("MapView Node could not be found");
```

Comment:
I think TestFX will thrown an exception if the node isn't found. So no need for `fail`.